lib/portus/json.rb

```diff
@@ -0,0 +1,32 @@
+module Portus
+  # A safe version of the ::JSON class. This class catches exceptions and logs
+  # them properly. This class assumes that the given object is a
+  # request/response.
+  class JSON
+    # Parses the body of the given request/response. Any exception is catched
+    # by this application and it's properly logged.
+    #
+    # Returns the given parsed JSON object on success, nil otherwise.
+    def self.parse(obj)
+      # Depending on whether this is a request or a response, the body will be
+      # a string or an object that responds to :read.
+      body = obj.body.is_a?(String) ? obj.body : obj.body.read
+      ::JSON.parse(body)
+    rescue ::JSON::ParserError => e
+      Rails.logger.warn "JSON: parser error!"
```

Comment:
Cronos only logs its own stuff: when has been run, the job being executed, etc. So, in my opinion everything should be logged into $environment.log, it's easier for the user to just check out one file, instead of a set of files.